packages/core/src/components/select/_select.scss

```diff
@@ -140,5 +147,22 @@
         color: rgba($ray-color-gray-60, 0.4);
       }
     }
+
+    &--simple {
+      flex: 1;
+      border: none;
+      background-color: transparent;
+      border-radius: 0;
+
+      &__input {
+        font-size: ($ray-field-label-size * 1.2);
```

Comment:
this is specifically to make the flags a but larger .. not sure if we should set this font size here